kubernetes/platform-services/values.yaml

```diff
@@ -204,6 +204,7 @@ attribute-service:
     limits:
       cpu: "0.5"
       memory: "256Mi"
+
```

Comment:
Looks like no change in this file, you can revert back to head.

Reply:
some service values had spaces between them and some didn't. This adds consistency and increases readability.